website/docs/docs/cloud/connect-data-platform/connect-starburst-trino.md

```diff
@@ -4,18 +4,18 @@ description: "Configure Starburst/Trino connection."
 sidebar_label: "Connect Starburst/Trino"
 ---
 
-The following are the required fields for setting up a connection with Starburst Enterprise, Starburst Galaxy, and Trino cluster. Unless specified, "cluster" will mean any of these products' clusters.     
+The following are the required fields for setting up a connection with a [Starburst Enterprise](https://docs.starburst.io/starburst-enterprise/index.html), [Starburst Galaxy](https://docs.starburst.io/starburst-galaxy/index.html), or [Trino](https://trino.io/) cluster. Unless specified, "cluster" means any of these products' clusters.
 
 | Field | Description | Examples |
 | --- | --- | --- |
 | **Host** | The hostname of your cluster. Don't include the HTTP protocol prefix. | `mycluster.mydomain.com` |
 | **Port** | The port to connect to your cluster. By default, it's 443 for TLS enabled clusters. | `443` |
-| **User** | The username (of the account) to log in to your cluster. When connecting to Starburst Galaxy clusters, you must include the role of the user as a suffix to the username.<br/><br/> | Format for Starburst Enterprise or Trino: <br/> <ul><li>`user.name`</li><li>`user.name@mydomain.com`</li></ul><br/>Format for Starburst Galaxy:<br/> <ul><li>`user.name@mydomain.com/role`</li></ul> |
+| **User** | The username (of the account) to log in to your cluster. When connecting to Starburst Galaxy clusters, you must include the role of the user as a suffix to the username.<br/><br/> | Format for Starburst Enterprise or Trino depends on configured authentication type. <br/>Format for Starburst Galaxy:<br/> <ul><li>`user.name@mydomain.com/role`</li></ul> |
 | **Password** | The user's password. |   |
 | **Database** | The name of a catalog in your cluster. | `my_postgres_catalog` |
 | **Schema** | The name of a schema in your cluster that exists within the specified catalog.  | `my_schema` |
```

Comment:
Is this optional? If yes, we should add that info. Also we should not use possessive example values .. how about just `example_schema`

Reply:
```suggestion
| **Schema** | The name of a schema in your cluster that exists within the specified catalog.  | `example_schema` |
```

Reply:
this is also required. thanks for checking, @mosabua !